src/cmds/upgrade.js

```diff
@@ -0,0 +1,56 @@
+const { execSync } = require('child_process')
+const { default: fetch } = require('node-fetch')
+const prompts = require('prompts')
+
+function compareVersions(current, latest) {
+  let currentArr = current.split('.')
+  let latestArr = latest.split('.')
+  for (let i = 0; i < 3; i++) {
+    let currentNum = Number(currentArr[i])
+    let latestNum = Number(latestArr[i])
+    if (currentNum > latestNum) return 1
+
+    if (latestNum > currentNum) return -1
+
+    if (!isNaN(currentNum) && isNaN(latestNum)) return 1
+
+    if (isNaN(currentNum) && !isNaN(latestNum)) return -1
+  }
+  return 0
+}
+
+function checkCurrentVersion() {
+  const command = "yarn global list --pattern @livestorm/cli | grep -oP '@livestorm/cli@\K\d+.\d+.\d+'"
+  return execSync(command)
+}
+
+function checkLatestVersion() {
+  const response = await fetch('https://registry.npmjs.org/@livestorm/cli')
+  const json = await response.json()
+  return json['dist-tags']['latest']
+}
+
+module.exports = () => {
+  try {
+    const current = checkCurrentVersion()
+    const latest = checkLatestVersion()
+    if (compareVersions(current, latest) !== -1) return false
+    
+    prompts({
+      type: 'text',
+      name: 'upgrade',
+      message: "We noticed your CLI isn't up to date, do you want to upgrade? (yes/no)",
+      validate: value => {
+        return (value !== 'no' || value !== 'yes')
+      }
+    }).then((answer) => {
```

Comment:
Would be great to not check at every run, maybe like once a day ? Or every X commands ?

Reply:
it looks excesive for me as well to run it on every command.
But what is the best way to track it?
To write some data in a temp file?

Reply:
You can use what @jinevo used here https://github.com/livestorm/livestorm-plugin-cli/blob/master/src/cmds/envs.js
```
const configStore = require('../helpers/configStore.js')
```